modules/account-lib/src/coin/eth2/keyPair.ts

```diff
@@ -32,6 +32,22 @@ export class KeyPair extends BlsKeyPair {
    * @returns {string} The address derived from the public key
    */
   getAddress(): string {
-    return this.getKeys().publicKey.toHexString();
+    return this.getKeys().pub;
+  }
+
+  static isValidPub(pub: string): boolean {
+    return BlsKeyPair.isValidBLSPub(pub);
+  }
+
+  static isValidPrv(prv: string | Buffer) {
+    if (typeof prv === 'string') {
+      return BlsKeyPair.isValidBLSPrv(prv);
+    }
+    try {
+      prv = '0x' + Buffer.from(prv).toString('hex');
+      return BlsKeyPair.isValidBLSPrv(prv);
```

Comment:
Nit: generally bad form to modify parameters. I would just directly pass `'0x' + Buffer...` into `BlsKeyPair.isValid..`, or use a new local variable